src/utils/index.js

```diff
@@ -4,11 +4,16 @@ import qs from 'query-string';
 
 import history from '../data/history';
 
-const getDateString = date => (date ? moment.utc(date).format('YYYY-MM-DD') : '');
+const dateWithDashes = date => (date ? moment.utc(date).format('YYYY-MM-DD') : '');
+const dateWithSlashes = date => (date ? moment.utc(date).format('YYYY/MM/DD') : '');
```

Comment:
Is there a reason to have all of this date with slashes if we really just want dashes everywhere? Why not just always use dashes, Safari or otherwise?

Reply:
the date is rendered now on the app with slashes
do you want to change everything to dashes?